serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,163 @@
+    }
+  ]
+----
+
+The `operation` property is composed by three tokens separated by #: the uri of the proto file, the name of the service and the name of the method. Kogito supports three URI schemes: htpp, file and classpath, the latter being the default. Therefore, in this particular case, the flow is declaring that greeting.proto is expected to be found in the project classpath.
+
+The flow execution consist of a single https://github.com/serverlessworkflow/specification/blob/main/specification.md#operation-state[operation] state, composed by one https://github.com/serverlessworkflow/specification/blob/main/specification.md#action-definition[action].
+
+[source,json]
+----
+ {
+    "name": "sayHello",
+    "functionRef" : {
+        "refName": "sayHello",
+        "arguments": {
+            "name": ".name",
+            "language": ".language"
+        }
+    }
+}
+----
+
+The action invokes `sayHello` rpc method using as  parameters two model variables: `name` and `language`, provided as part of the REST call that starts the flow. 
+ 
+=== Application properties
+
+We need to tell the application where the grpc server is located
+
+[source,properties]
+----
+quarkus.grpc.clients.Greeter.host=localhost
+quarkus.grpc.clients.Greeter.port=50051
+----
+
+Internally, Kogito Serverless workflow implementation use the exact set of properties than https://quarkus.io/guides/grpc-service-consumption#client-configuration[Quarkus grpc client].
+The "client-name" used by kogito is the service name declared in the proto file. `Greeter` in this case
+
+Why port is 50051?. In the same repo than the workflow application, there is available a https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-server-rpc-quarkus[maven project] that provides a simple implementation of the greeter service. By default, this gRPC server runs on port 50051, hence the value for the port property. 
+
+=== Maven POM 
+
+gRPC functionality is built within https://github.com/kiegroup/kogito-runtimes/tree/main/quarkus/extensions/kogito-quarkus-serverless-workflow-extension[Serverless Workflow Quarkus extension], which means that there is not need to include specific dependencies for gRPC to work with Serverless workflow. 
+The same set of dependencies that are included for the Hello Workflow example will work. 
+
+One section of the pom that deserve mention is this one
+
+[source,xml]
+----
+<resource>
+ <directory>${project.basedir}/../serverless-workflow-greeting-server-rpc-quarkus/src/main/proto</directory>
+ <includes>
+   <include>greeting.proto</include>
+  </includes>
+</resource>
+----
+
+Remember that the URI of the proto file specified in the workflow is expecting the proto file to be in the project classpath. This snippet ensures the proto file defined in the grpc server project is included in the classpath of the workflow application. 
```

Comment:
```suggestion
Remember that the URI of the proto file specified in the workflow is expecting the proto file to be in the project's classpath. Therefore, this snippet ensures the proto file defined in the gRPC server project is included in the classpath of the workflow application. 
```

Reply:
Shouldn't we in general be explaining more how people can _add_ a gRPC call to their workflow, rather than just explaining how the example works?  So in that case we would be describing you need to add this to your pom.xml?